src/main/java/org/mozilla/jss/CryptoManager.java

```diff
@@ -528,8 +549,40 @@ public static synchronized void initialize( InitializationValues values )
                     "Must set ocspResponderCertNickname");
             }
         }
+        //Right now, allow only the choice of a full NSS init or a NSS context init, usefule for libraries,
```

Comment:
please fix the tab and the spelling of "useful" ;-)

Reply:
Done